packages/template-manager/internal/server/create_template.go

```diff
@@ -63,79 +52,34 @@ func (s *serverStore) TemplateCreate(templateRequest *template_manager.TemplateC
 		BuildLogsWriter: logsWriter,
 	}
 
-	buildStorage := s.templateStorage.NewBuild(template.TemplateFiles)
```

Comment:
logic for template build extracted from grpc handler to template builder service